README.md

```diff
@@ -1,2 +1,7 @@
+
+llol
+
+Reetik was Here™
```

Comment:
No you weren't

Reply:
APROVE MY PULL REQUEST SCRUB

Reply:
Also it's called staging not adding

Reply:
Okay but that makes it sound more complicated so sorry

Reply:
not really, your on stage now your off stage

Reply:
okay sorry geez